### PR TITLE
Modify the way of starting netperf_stress in netstress_kill_guest

### PR DESCRIPTION
**Pull request by fbq815**
Virtual_network: Since this case has an abnormal activity for killing
guests, change the way of starting netperf_stress

ID: 2015540

Signed-off-by: Boqiao Fu <bfu@redhat.com>

### CODE REVIEW

**Comment by fbq815** [comment]
Test result:
JOB ID     : 07324a9897dbe21e4b2b18ab740d75e7d5c640fa
JOB LOG    : /root/avocado/job-results/job-2021-11-04T03.49-07324a9/job.log
 (1/1) Host_RHEL.m9.u0.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.s390x.io-github-autotest-qemu.netstress_kill_guest.load.s390-virtio: PASS (539.41 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado/job-results/job-2021-11-04T03.49-07324a9/results.html
JOB TIME   : 539.84 s

**Comment by fbq815** [comment]
@PaulYuuu @ngu-niny  please help me review this patch

File: generic/tests/netstress_kill_guest.py [hunk 1]
```diff
@@ -52,6 +54,72 @@ def kill_and_check(test, vm):
...
+                test.error("Can not start netperf client.")
+            start_time = time.time()
+            stop_time = time.time()
+            run_time = stop_time - start_time
+            if n_client.is_netperf_running():
+                test.fail("netperf still running,netperf hangs")
+            elif test_duration - 5 <= run_time:
+                logging.info("netperf runs successfully")
+            else:
+                test.fail("netperf terminated unexpectedly,executed %ss" % run_time)
```
Comment (PaulYuuu) [inline generic/tests/netstress_kill_guest.py:117]: For this case, seems we don't need to take care of the status of netperf, you can delete them and only ensure that netperf is running(netperf_test_duration)

Comment (fbq815) [reply generic/tests/netstress_kill_guest.py]: solved

**Comment by fbq815** [comment]
Test result:
JOB ID     : c81c6383c1bf3067d0f40bb2ef0aeaabd71ba603
JOB LOG    : /root/avocado/job-results/job-2021-11-09T02.04-c81c638/job.log
 (1/1) Host_RHEL.m9.u0.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.s390x.io-github-autotest-qemu.netstress_kill_guest.load.s390-virtio: PASS (546.50 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado/job-results/job-2021-11-09T02.04-c81c638/results.html
JOB TIME   : 546.84 s

**Comment by fbq815** [comment]
@PaulYuuu @ngu-niny please help me review this patch

**Comment by PaulYuuu** [comment]
LGTM, but the commit title and message do not match what you have changed.

**Comment by fbq815** [comment]
> LGTM, but the commit title and message do not match what you have changed.

Modified, how do you think about this time?

**Comment by fbq815** [comment]
Test result on RHEL8:
JOB ID     : 21b92d77cd5774436c7e00b2a585c9e6f4980a83
JOB LOG    : /root/avocado/job-results/job-2021-11-15T02.50-21b92d7/job.log
 (1/1) Host_RHEL.m8.u6.product_rhel.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.netstress_kill_guest.load.s390-virtio: PASS (549.77 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado/job-results/job-2021-11-15T02.50-21b92d7/results.html
JOB TIME   : 550.22 s

**Comment by PaulYuuu** [comment]
> > LGTM, but the commit title and message do not match what you have changed.
> 
> Modified, how do you think about this time?

Seems not related to `do not report debug.log`

**Comment by fbq815** [comment]
> > > LGTM, but the commit title and message do not match what you have changed.
> > 
> > 
> > Modified, how do you think about this time?
> 
> Seems not related to `do not report debug.log`

does it seems better this time?

**Review by PaulYuuu — APPROVED** [review]

**Comment by ngu-niny** [comment]
LGTM.

Acked-by: Nini Gu <ngu@redhat.com>

**Pull request merged**